include/nuttx/fs/fs.h

```diff
@@ -298,6 +305,11 @@ struct mountpt_operations
             size_t buflen);
   off_t   (*seek)(FAR struct file *filep, off_t offset, int whence);
   int     (*ioctl)(FAR struct file *filep, int cmd, unsigned long arg);
+  int     (*truncate)(FAR struct file *filep, off_t length);
+  int     (*mmap)(FAR struct file *filep, FAR struct mm_map_entry_s *map);
+  int     (*munmap)(FAR struct task_group_s *group,
+            FAR struct mm_map_entry_s *map, void *start,
+            size_t length);
```

Comment:
don't need start/length since mm_map_entry_s already has these fields

Reply:
yes, you do. "map" is the existing mapping, start and length indicate the area which user want's to unmap. These are not necessarily the same; driver may support also unmapping a part of the previously mapped area.

Reply:
Ok, if you want to suppor the partial unmap, the argument is required.